source/Command.h

```diff
@@ -29,6 +30,10 @@ class DataNode;
 // everything the AI wants a ship to do, or all keys the player is holding down.
 class Command {
 public:
+	// This must match the maximum command number in Command.cpp or the build will fail.
+	static const unsigned COMMAND_COUNT = 33;
```

Comment:
Does this need to be public?

Reply:
No. That was an oversight.

Reply:
Apparently, the answer is, "yes, that needs to be public" otherwise the COMMAND_COUNT is not available to the static_assert in MakeCommandState in Command.cpp. I could work around that by moving MakeCommandState to Command.h, but I'm trying to avoid revealing templates at the .h level since it makes Hurleveur sad.

Reply:
It is private now, and it uses state.size()